playwright-tests/tests/intents/intents-payment-request-detail.spec.js

```diff
@@ -1,11 +1,34 @@
 import { test } from "../../util/test.js";
 import { expect } from "@playwright/test";
-import { redirectWeb4, getLocalWidgetContent } from "../../util/web4.js";
+import { redirectWeb4 } from "../../util/web4.js";
+
+test.use({
+  viewport: {
+    width: 1280,
+    height: 800,
+  },
+});
+
+test.beforeEach(async ({ page }, testInfo) => {
+  // Skip all tests in this file if not running on the treasury-testing project
+  test.skip(
+    testInfo.project.name !== "treasury-testing",
+    "These tests only run on the treasury-testing project"
+  );
```

Comment:
why is that?

Reply:
Since the test is using the webassemblymusic-treasury as instance / dao it would be the same for all 3 projects, and so it doesn't make sense to run the same test 3 times ( and also fail because of rate limits then ).